src/Extension/WithLastModified.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Twig\Extension;
+
+interface WithLastModified
```

Comment:
This is also a new feature required to detect when the class that defines attributes have been updated.

Reply:
@stof I can't replace the modification date by a has, the `Environment::isFresh` relies on a date comparison between the ExtensionSet and the cache file mtime.

https://github.com/twigphp/Twig/blob/aeeec9a5e907a79e50a6bb78979154599401726e/src/Environment.php#L416-L419

Reply:
`LastModifiedAwareInterface`? What's the existing convention in Twig?

Reply:
Renamed to `ModificationAwareInterface`. The convention if to suffix with `Interface`.

Reply:
With the last change, I removed this interface. `ExtensionSet` is now aware of the extension classes that use attributes, and check their modification date as well.